include/deal.II/numerics/data_out_rotation.h

```diff
@@ -118,29 +118,16 @@ namespace internal
  *
  * @ingroup output
  */
-template <int dim, typename DoFHandlerType = DoFHandler<dim>>
-class DataOutRotation
-  : public DataOut_DoFData<DoFHandlerType, DoFHandlerType::dimension + 1>
+template <int dim, int spacedim = dim>
+class DataOutRotation : public DataOut_DoFData<dim, dim + 1, spacedim, dim + 1>
```

Comment:
I suspect that this ought to be 
```suggestion
class DataOutRotation : public DataOut_DoFData<dim, dim + 1, spacedim, spacedim + 1>
```
but that too is for a later patch.